examples/meta_example.py

```diff
@@ -0,0 +1,36 @@
+# # Meta-analysis: an example of use of the KDA & MKDA method.
+
+# +
```

Comment:
what tool are you using to transform scripts into notebooks?

Reply:
I'm using Jupytext with the light script format. But I'm considering to use the percent script format instead since it can be used to run a notebook in vs code.